### ISSUE TITLE
Content build action ignoring .mgcb output path settings

### ISSUE THREAD

**Issue by Ben-Hurr** [mono/MonoGame#4951]
When building a monogame project from VS2015 (on windows 7, anyway), your build files and finished .xnb's etc. get dropped into a hokey default _'Content'_ and _'bin/(whatever platform)_ folder, rather than what you specified in the actual .mgcb file.

Building from the pipeline GUI or command line however, works as expected, putting your files where you tell it to.

Not that much of an issue if you never play with the default path settings, but if you don't want your game's resource folder to be hard-coded as _'Content'_ it gets troublesome.

**Comment by cra0zy** [comment]
> When building a monogame project from VS2015 (on windows 7, anyway), your build files and finished .xnb's etc. get dropped into a hokey default 'Content' and 'bin/(whatever platform) folder, rather than what you specified in the actual .mgcb file.

This is normal, bin and obj paths in the .mgcb file are for when you build the file with the MonoGame Pipeline Tool. I think you should be able to change the default path by moving the Content.mgcb file to the desired folder structure (tho I've never tried it myself).

**Comment by tomspilman** [comment]
> This is normal, bin and obj paths in the .mgcb file are for when you build the file with the MonoGame Pipeline Tool.

Yeah...  it is as designed.  Still being one of the people that designed it... I hate it too.

We need to figure out a nicer way to do that, but I don't think it can happen until we can support per-platform settings in the .mgcb file.
